src/notifications/screens/notifications.screen.js

```diff
@@ -344,19 +353,27 @@ class Notifications extends Component {
             this.notifications().length === 0 &&
             <LoadingContainer
               animating={this.isLoading() && this.notifications().length === 0}
-              text={`Retrieving ${type === 0
-                ? 'unread'
-                : type === 1 ? 'pending' : 'all'} notifications`}
+              text={translate(
+                'notifications.main.retrievingMessage',
+                language,
+                {
+                  notificationType:
+                    type === 0
+                      ? 'unread'
+                      : type === 1 ? 'participating' : 'all',
+                }
+              )}
               style={styles.marginSpacing}
             />}
 
           {!this.isLoading() &&
             this.notifications().length === 0 &&
             <View style={styles.textContainer}>
               <Text style={styles.noneTitle}>
-                {`You don't have any${type === 0
-                  ? ' unread'
-                  : type === 1 ? ' pending' : ''} notifications!`}
+                {translate('notifications.main.noneMessage', language, {
+                  notificationType:
+                    type === 0 ? ' unread' : type === 1 ? ' participating' : '',
```

Comment:
Cheers, fixed it by just simplifying the message and removing the interpolation.